frontend/src/components/NavigationItems.vue

```diff
@@ -1,96 +1,33 @@
-
 <template>
-  <b-nav fill>
+  <div>
     <b-nav-item
       v-for="link in links"
       :to="link.to"
-      :key="link.text">{{ link.text }}</b-nav-item>
-
-    <Language />
-
-    <a
-      class="out-link"
-      href="http://codeforpoznan.pl"
-      target="_blank">
-      <img
-        class="out-link-image"
-        src="../assets/logo_codeforpoznan.svg" >
-    </a>
-
-    <b-nav-item
-      v-if="user"
-      @click="LOGOUT"
-      to="/logout"
-      key="logout"
-      class="username"
-      :class="{ offline: !isOnline }"
-    >
-      {{ $t('common.logout') }}
-      <p>{{ user.username }}</p>
+      :key="link.text">
+      {{
+        $t(link.text)
+      }}
     </b-nav-item>
-  </b-nav>
+  </div>
 </template>
 
 <script>
-import { mapActions, mapGetters, mapState } from 'vuex';
-import * as actions from '../store/actions';
+import flatMap from 'array.prototype.flatmap'
```

Comment:
As we updated to core-js v3 - is another polyfill needed? If it is, everything is ok

Reply:
I couldn't get core-js to work 😞. So that's why I'm importing this